frontend/src/app/ReusableComponents/UserAuth.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
```

Comment:
nit: the folder name could probably be renamed to something like
`shared/components`